velox/functions/sparksql/Arithmetic.h

```diff
@@ -262,6 +262,16 @@ struct Log1pFunction {
   }
 };
 
+
+struct Expm1Function {
+  FOLLY_ALWAYS_INLINE void call(double& result, double a) {
+    // The std::expm1 is more accurate than the expression std::exp(num) - 1.0 if num is close to zero.
+    // Spark use java StrickMath we need to keep it as below.
```

Comment:
use -> uses

java -> Java's

typo in "StrickMath" ?

> Spark use java StrickMath we need to keep it as below.

typos... please, revisit

Reply:
Thanks for checking and all are resolved.